docs/cubeviz/export_data.rst

```diff
@@ -41,7 +41,7 @@ To get all subsets from the spectrum viewer:
 
 .. code-block:: python
 
-    subset1_spec1d = cubeviz.app.get_subsets_from_viewer("spectrum-viewer")
+    subset1_spec1d = cubeviz.app.get_subsets()
```

Comment:
```suggestion
    subset1_spec1d = cubeviz.specviz.app.get_subsets()
```
This will return all the subsets that specviz handles, which is the same as all the subsets from the spectrum viewer. You can maybe reword the above as "all spectral subsets" instead of "from the spectrum viewer", although both phrases are technically true.

Reply:
Ahh got catch. I was supposed to add `spectral_only=True` but in this context, I think your suggestion makes sense